Control Work 2/Task1/Task1/Client.cs

```diff
@@ -0,0 +1,36 @@
+﻿namespace ConsoleChat;
+
+using System.Net;
+using System.Net.Sockets;
+
+/// <summary>
+/// Console chat client realization
+/// </summary>
+public class Client : ConsoleChatFunctions
+{
+    private IPAddress ip;
+    private int port;
+
+    public Client(string host, int port)
+    {
+        ip = IPAddress.Parse(host);
+        this.port = port;
+    }
+
+    /// <summary>
+    /// Set connection between port on server and client
+    /// </summary>
+    public async Task Connect()
+    {
+        Console.WriteLine("Connecting to server ...");
+        using (var client = new TcpClient())
```

Comment:
```suggestion
        using var client = new TcpClient();
```
И дальше без лишнего отступа